nutils/transformseq.py

```diff
@@ -336,6 +336,10 @@ def get_evaluable(self, index: evaluable.Array) -> EvaluableTransformChain:
     return _EvaluableTransformChainFromSequence(self, index)
 
   def evaluable_index_with_tail(self, chain: EvaluableTransformChain) -> Tuple[evaluable.Array, EvaluableTransformChain]:
+    if isinstance(chain, _EvaluableTransformChainFromSequence) and chain._sequence == self:
```

Comment:
Would it be more natural to make `evaluable_index_with_tail` a method of EvaluableTransformChain rather than Transforms? The (two) special cases seem to depend on the value of `chain`, and the base implementation doesn't take anything from `self` beyond its length and `ndims`.

Reply:
Done. New method is `EvaluableTransformChain.index_with_tail_in`.